apps/minifront/src/components/v2/dashboard-layout/assets-card-title.tsx

```diff
@@ -3,23 +3,27 @@ import { CharacterTransition } from '@repo/ui/CharacterTransition';
 import { Dialog } from '@repo/ui/Dialog';
 import { Text } from '@repo/ui/Text';
 import { Info } from 'lucide-react';
+import { useId } from 'react';
 
-export const AssetsCardTitle = () => (
-  <div className='flex items-center gap-2'>
-    <CharacterTransition>Asset Balances</CharacterTransition>
-    <Dialog>
-      <Dialog.Trigger asChild>
-        <Button icon={Info} iconOnly='adornment'>
-          Info
-        </Button>
-      </Dialog.Trigger>
-      <Dialog.Content title='Asset Balances'>
-        <Text>
-          Your balances are shielded, and are known only to you. They are not visible on chain. Each
-          Penumbra wallet controls many numbered accounts, each with its own balance. Account
-          information is never revealed on-chain.
-        </Text>
-      </Dialog.Content>
-    </Dialog>
-  </div>
-);
+export const AssetsCardTitle = () => {
```

Comment:
You may want to review this file with whitespace changes hidden. I just added a `useId()` call, and then passed it to the dialog trigger and content to make the dialog "grow out of" the info icon.